src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private String color;
+
+    public abstract double getArea();
+
```

Comment:
It would be best if you had the interface for the `getArea()` method
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
    private String color;
```